libraries/sqs/lib_test/test_lwt.ml

```diff
@@ -5,7 +5,8 @@ module T = TestSuite(struct
 
     let access_key = Unix.getenv "AWS_ACCESS_KEY"
     let secret_key = Unix.getenv "AWS_SECRET_KEY"
+    let region = Unix.getenv "AWS_DEFAULT_REGION"
 
-    let run_request = Aws_lwt.Runtime.run_request ~access_key ~secret_key
+    let run_request x = Aws_lwt.Runtime.run_request ~access_key ~secret_key ~region x
```

Comment:
Curiously this `x` binding is required so that the function / module type checks.